docs-asciidoc/Makefile

```diff
@@ -35,6 +35,8 @@ html-author-mode:
 apidocs:
 	cd ${root_dir} && sbt java-support/doc
 	cp -r ${root_dir}/java-support/target/api ${site_dir}/docs/current/java/api
+	cd ${root_dir}/node-support && npm i
```

Comment:
Sure, let's add the install. The CI scripts currently do this in a before script, so the old docs setup requires dependencies to already have been installed, which is why not added here originally.